lib/resilient.js

```diff
@@ -153,8 +153,16 @@ Resilient.prototype.balancer = function (options) {
 }
 
 Resilient.prototype.send =
-Resilient.prototype.request = function (path, options, cb) {
-  return this._client.send(path, options, cb)
+Resilient.prototype.request = function (path, options) {
+  return new Promise( function(resolve, reject) {
+      this._client.send(path, options, function(err, res) {
+        if (err) {
+          reject(err);
+        } else {
+          resolve(res);
+        }
+      }.bind(this))
```

Comment:
This should be implemented in `client.send`, as it's the unified entry point for HTTP requests. We can cover all interface usage vectors from there.